.stylelintignore

```diff
@@ -1 +1,2 @@
+public/css/skeleton.css
```

Comment:
Isn't that already covered by the minified version (`skeleton.min.css`)?

Reply:
It wasn't passing for me when I tried it, as they are two different files. Do you not get the error on your end?

Reply:
Well, they are the exact same apart from the fact that the `*.min.css` are the `minify`ed versions.